src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -81,7 +83,61 @@ public function __construct(AuditManager $auditManager)
 
     public function getSubscribedEvents()
     {
-        return array(Events::onFlush, Events::postPersist, Events::postUpdate);
+        return array(Events::onFlush, Events::postPersist, Events::postUpdate, Events::postFlush);
+    }
+
+    public function postFlush(PostFlushEventArgs $eventArgs)
+    {
+        $em = $eventArgs->getEntityManager();
+
+        $uow = $em->getUnitOfWork();
+
+        //well, this is awkward, it's a reflection again
+        $uowR = new \ReflectionClass($uow);
+        $extraUpdatesR = $uowR->getProperty('extraUpdates');
+        $extraUpdatesR->setAccessible(true);
+
+        $extraUpdates = $extraUpdatesR->getValue($uow);
+
+        foreach ($extraUpdates as $update) {
+            list($entity, $changeset) = $update;
+
+            if ($this->metadataFactory->isAudited(get_class($entity))) {
```

Comment:
that's not relevant but early exits are much easy to read.

```php
if (!$this->metadataFactory->isAudited(get_class($entity))) {
    continue;
}
```

thus you havn't deep nesting. the same below.

```php
if (!isset($updateData[$meta->table['name']]) || !$updateData[$meta->table['name']]) {
    continue;
}
```